### PR TITLE
two stage build Dockerfile added

### PR DESCRIPTION
**Pull request by sedoy26**

### CODE REVIEW

File: Dockerfile.test [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    jwcrypto>=0.9.1 \
+    fastjsonschema \
+    passlib \
+    bcrypt \
+    python-ldap \
+    jinja2 \
+    pyotp \
+    webauthn \
+    git+https://github.com/TeskaLabs/asab.git
+# && apk del buildenv
```
Comment (byewokko) [inline Dockerfile.test:45]: Remove this line pls, since there is no buildenv anymore.

File: Dockerfile.test [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+COPY ./.git /app/seacat-auth/.git
+RUN asab-manifest.py ./MANIFEST.json
+
+
+FROM alpine:3.13
+
+
+RUN apk add --no-cache \
+  python3 
+#   py3-pip
```
Comment (byewokko) [inline Dockerfile.test:61]: This doesn't have to be here either.

**Review by byewokko — CHANGES_REQUESTED** [review]
@sedoy26 Could you please add the image sizes before and after, just for information? Just add it to the PR description or comment.

**Comment by ateska** [comment]
As discussed, we need some hard numbers here - size of images + size of the base image (alpine).

**Comment by sedoy26** [comment]
@byewokko @ateska

**Comment by sedoy26** [comment]
@byewokko @ateska 
original image size: 238MB
two stage image size: 218MB
alpine image size: 5.61MB

**Comment by ateska** [comment]
Hmms, not great, not terrible.
Let's merge this, we can investigate later how to reduce the size futher if needed.

**Review by byewokko — APPROVED** [review]

**Pull request merged**